client/app/src/main/java/uk/ac/cam/cl/echo/extrusionfinder/client/servercopies/configuration/Configuration.java

```diff
@@ -0,0 +1,58 @@
+package uk.ac.cam.cl.echo.extrusionfinder.client.servercopies.configuration;
+
+public class Configuration {
+    public static final String MONGO_HOST = "localhost";
```

Comment:
It's an exact copy of the file on the server, or at least as close as it can be. This is to simplify updating it if the server copy changes.